src/main/java/Duke.java

```diff
@@ -1,10 +1,260 @@
+import java.util.Scanner;  // Import the Scanner class
```

Comment:
There are some areas of improvement on this file: 
1. Everything is squeezed into one single main methods. That's too long. Try to refractor them into multiple methods, or even into other classes.
2. Arrowhead code. Generally we try not to have more than 3 levels of indentation in our code. Again, refactoring may help in this.